components/layout/display_list_builder.rs

```diff
@@ -2135,7 +2135,7 @@ impl FragmentDisplayListBuilding for Fragment {
                                        .get_inheritedtext()
                                        ._servo_text_decorations_in_effect;
         // Note that the text decoration colors are always the same as the text color.
-        text_decorations.underline = text_decorations.underline.map(|_| text_color);
+        text_decorations.underline = text_decorations.underline;//.map(|_| text_color);
```

Comment:
Why is this correct? Mind leaving a comment so people note it's intentional?

Also, if this change is really correct, please remove the leftover comment.